docs/_plugins/search_index.rb

```diff
@@ -181,7 +189,7 @@ def execute
   end
 
   model = {
-    id: post.url,
+    id: "#{post.data['name']}_#{post.data['edition']}_#{post.data['language']}",
```

Comment:
It is not really matter in this case, but we use a different order in other places, like model filenames:
- name
- language
- edition

I think it will be better for supporting in the future. Let's change the order.

Reply:
Done